src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -119,24 +155,39 @@ public void periodic() {
                 driveStates = DriveStates.FIELD_ABSOLUTE;
                 System.out.println("FIELD Relative OFF");
             }
-        }
+        } else if (driveStates == DriveStates.TELEOP_ALIGNING) {
+            driveToPosePID(robot.manager.targetPose);
+            if (nearSetPose(robot.manager.targetPose) && robot.manager.state != ManagerStates.SCORING_AMP) {
```

Comment:
you could probably just set the manager state to scoring amp and transition drive back to seb control instantly, instead of waiting. cleaner code, and seb might be able to start driving slightly faster since amp score is a little slow.